lib/ts/types.ts

```diff
@@ -30,14 +30,15 @@ export type SuperTokensConfig = {
     /*
      * List of recipes for authentication and session management.
      */
-    recipeList: CreateRecipeFunction<any, any, any>[];
+    recipeList: CreateRecipeFunction<any, any, any, any>[];
 };
 
 export type CreateRecipeFunction<
     Action,
     PreAPIHookContext extends RecipePreAPIHookContext<Action>,
-    Config extends RecipeConfig<Action, PreAPIHookContext>
-> = (appInfo: NormalisedAppInfo) => RecipeModule<Action, PreAPIHookContext, Config>;
+    PostAPIHookContext extends RecipePostAPIHookContext<Action>,
```

Comment:
We can Make this just `CreateRecipeFunction<Action>`?

Reply:
Done